exercises/palindrome-products/canonical-data.json

```diff
@@ -121,9 +121,7 @@
               "min": 1002,
               "max": 1003
             },
-            "expected": {
-                "error": "no palindrome with factors in the range 1002 to 1003"
-            }
+            "expected": []
```

Comment:
As it stands, this doesn't have the same type as the cases that do result in palindromes. This proposal has an **array**. Previous cases have an **object** with an integer `value` and array `factors`.

Using an array here is hiding the fact that we have no `value`, and I predict that not being explicit about this will confuse readers of this file.

Therefore, I suggest if we are going to go this route, it should be:

```json
{
    "value": null,
    "factors": []
}
```